spec/controllers/projects_controller_spec.rb

```diff
@@ -26,11 +27,15 @@
       end
 
       context 'rendering the show' do
-        before :each do
-          post :create, :project => subject_params
-        end
 
+        subject { post :create, :project => subject_params }
+
+        it 'should increase total number of project_attributes' do
+          expect{subject}.to change{@user.project_attributes.count}.by(1)
```

Comment:
Instead of touching the database, what do you think about creating mocks and asserting they are called? If need help with the mocking concepts, please let me know.